### PR TITLE
docs: update and improve contributing guidelines

### LINKED ISSUE #147: Contribution Guide Additions
Add information to `.github/CONTRIBUTING.md` regarding:
- Linting (#115)
- Not adding any classes or ids to CSS files
- Theme color guidelines:
  - Reuse existing colors where possible and only for the purpose that their variable describes (eg. don't use text variables for background colorss)
  - Add new colors to both theme files if necessary
  - Don't add colors directly to the css files
  - How to create your own custom theme file (like #42 had done for a "community" theme)

A pull request could be made for each individual item!

### PR DESCRIPTION
**Pull request by jonaskuske**
Anything missing?

close #147

### CODE REVIEW

File: .github/CONTRIBUTING.md [hunk 1]
```diff
@@ -4,67 +4,126 @@ Water.css becomes better for everyone when people like you help make it better!
...
+
+<br>
+
+> **TL;DR**:
+> 1. `yarn install` – install dependencies
+> 2. `yarn dev` – start the dev server, make your changes
+> 3. `yarn validate` – make sure your changes pass all tests
+> 4. `yarn changeset` – describe your changes for the changelog
+
+<br>
```
Comment (kognise) [inline .github/CONTRIBUTING.md:27]: I love the toc

File: .github/CONTRIBUTING.md [hunk 2]
```diff
@@ -4,67 +4,126 @@ Water.css becomes better for everyone when people like you help make it better!
...
 ```
 yarn lint
 ```
 
-**Changelogs**
+<details>
+  <summary>ℹ A note about puppeteer and WSL</summary>
+  <br>
+  <blockquote>
+    The accessibility checks use puppeteer, a tool that uses Chrome to render websites "headlessly", without a visible interface. In some environments like the <a href="https://aka.ms/wsl">Windows Subsystem for Linux</a>, you'll need to manually configure and run an X-Server in order for puppeteer to work.
```
Comment (kognise) [inline .github/CONTRIBUTING.md:108]: Ooh, I didn't realize this!

Comment (jonaskuske) [reply .github/CONTRIBUTING.md:108]: Yeah, right now you need to run VcXsrv or a similar X-Server in Windows and point the `DISPLAY` env variable in the Linux environment at it.

But Microsoft announced at Build a couple days ago that graphical apps and GPU access is coming to WSL, so I assume they'll add a native X-Server integration then :)

**Comment by kylejrp** [comment]
This is seriously awesome work @jonaskuske! Great job!

**Pull request merged**